concourse/scripts/build_diskquota.sh

```diff
@@ -11,7 +11,9 @@ function pkg() {
     fi
 
     pushd /home/gpadmin/diskquota_artifacts
-    cmake /home/gpadmin/diskquota_src
+    local last_release_diskquota
+    last_release_diskquota=$(readlink -e /home/gpadmin/last_released_diskquota_bin/diskquota-*.tar.gz)
+    cmake /home/gpadmin/diskquota_src -DDISKQUOTA_PREVIOUS_INSTALLER="${last_release_diskquota}"
```

Comment:
```suggestion
    cmake /home/gpadmin/diskquota_src -DDISKQUOTA_LAST_RELEASE="${diskquota_last_release}"
```
Would this be more clear?

Reply:
changed to `DISKQUOTA_LAST_RELEASE_PATH`